src/components/MessageBanner/MessageBanner.md

```diff
@@ -0,0 +1,45 @@
+# List
+Presents a message to the user, with an optional call to action.
```

Comment:
It might be worth mentioning that these can be errors, updates, or alerts (just to help explain what types of messages can go here).